recipes/frogs/meta.yaml

```diff
@@ -1,53 +1,62 @@
 {% set name = "frogs" %}
-{% set version = "2.0.1" %}
+{% set version = "3.1.0" %}
+{% set phyloseq_extended_version = "0.99" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 3
+  number: 0
   skip: true  # [py3k]
 
+# 1) Uplaod FROGS source code
+# 2) Add phyloseq-extented lib from Mahendra Mariadassou to FROGS (lib/external-lib) :
 source:
-  url: https://github.com/oinizan/FROGS/archive/v{{ version }}.tar.gz
-  md5: da41b0950a750be2038cefa717edf9df
+  - url: https://github.com/geraldinepascal/FROGS/archive/v{{ version }}.tar.gz 
+    md5: 1a72f44e82517b6839a82cd677b98858 
+  - url: https://github.com/mahendra-mariadassou/phyloseq-extended/archive/v{{ phyloseq_extended_version }}.tar.gz
+    md5: 1ec072751503eebb791e876a60433120
+    folder: lib/external-lib
 
 requirements:
   host:
     - python
-    - rdptools >=2.0.2
+
   run:
+    - emboss 6.6.0
     - python
-    - scipy >=0.17.1
+    - scipy 1.1.0
     - perl
-    - perl-io-gzip >=0.20
-    - flash >=1.2.11
-    - cutadapt >=1.8.3
-    - swarm >=2.1.13
-    - vsearch >=2.4.3
-    - blast >=2.2.31
-    - rdptools >=2.0.2
+    - perl-io-gzip 0.20
+    - flash 1.2.11
+    - cutadapt 1.18
+    - swarm 2.2.2
+    - vsearch 2.9.1
+    - blast 2.7.1
```

Comment:
Those tight pins can make it difficult to find a solution later on. Wouldn't e.g. `blast 2.7.*` suffice?

Reply:
We have chosen to pin **all** the dependencies version to ensure we will **always** have the same results with this version of FROGS